src/main/java/org/myrobotlab/service/Servo.java

```diff
@@ -548,14 +544,14 @@ public synchronized void moveTo(double pos) {
 
     lastActivityTime = System.currentTimeMillis();
 
-    if (lastPos != pos) {
+   // if (lastPos != pos) {

```

Comment:
this introduces a slight change in behavior for the auto-disable  (probably warranted.. just calling out that there might be some testing to do around auto-disable/enable functionality here.)

Reply:
honestly I don't understand how this worked previously .. pos is not guaranteed to be set - what value is this ?